cmd/cloud-controller-manager/BUILD

```diff
@@ -16,14 +16,29 @@ go_binary(
 
 go_library(
     name = "go_default_library",
-    srcs = ["controller-manager.go"],
+    srcs = [
+        "main.go",
+        "nodeipamcontroller.go",
+    ],
     importpath = "k8s.io/cloud-provider-gcp/cmd/cloud-controller-manager",
     deps = [
-        "//cmd/cloud-controller-manager/app:go_default_library",
+        "//vendor/github.com/spf13/pflag:go_default_library",
+        "//vendor/k8s.io/apiserver/pkg/util/feature:go_default_library",
+        "//vendor/k8s.io/cloud-provider:go_default_library",
+        "//vendor/k8s.io/cloud-provider/app:go_default_library",
+        "//vendor/k8s.io/cloud-provider/app/config:go_default_library",
+        "//vendor/k8s.io/cloud-provider/options:go_default_library",
+        "//vendor/k8s.io/component-base/cli/flag:go_default_library",
         "//vendor/k8s.io/component-base/logs:go_default_library",
         "//vendor/k8s.io/component-base/metrics/prometheus/clientgo:go_default_library",
         "//vendor/k8s.io/component-base/metrics/prometheus/version:go_default_library",
+        "//vendor/k8s.io/controller-manager/app:go_default_library",
+        "//vendor/k8s.io/klog/v2:go_default_library",
+        "//vendor/k8s.io/kubernetes/pkg/controller/nodeipam:go_default_library",
```

Comment:
I get it but yuck. Please make sure we have an issue filed to move these to staging, so we can stop depending on K/K internal.

Reply:
Working on it. :)